package.json

```diff
@@ -34,10 +34,13 @@
     "@biconomy/account": "^4.1.1",
     "ethereumjs-tx": "^2.1.2",
     "ethers": "^5.7.2",
-    "tslib": "^2.6.2"
+    "sinon": "^18.0.0",
```

Comment:
i understand sinon is needed for your testcase, we can't add new dependencies to sdk especially the ones we can do without, we need to remove sinon and your testcase now that your it runs without errors. viem has been updated as well to "viem": "^2.12.0" on my recent update

Reply:
Can I still use sinon in my tests without adding in package.json.